src/odemis/driver/xt_client.py

```diff
@@ -2065,6 +2093,10 @@ def stop(self, axes=None):
             logging.exception("Unexpected failure when updating position")
 
 
+# Value to use on the FASTEM to activate the immersion mode
+COMPOUND_LENS_FOCUS_IMMERSION = 2.0
```

Comment:
I would usually put constants at the top of the module, why put it here?

Reply:
Sometimes when a constant is only used by one class, I place it just on top of that class. I don't have a strong preference though, so I'll move it higher.